azuredeploy.json

```diff
@@ -193,7 +205,7 @@
                 400,
                 800
             ],
-            "defaultValue": 50,
+            "defaultValue": 100,
```

Comment:
This and subsequent changes in DB SKU increase the cost of an initial deployment. I assume the changes are based on results from load testing and we will, eventually, provide T-Shirt sizing (#36) so these changes are probably OK. However, does they really belong in this PR? Do they have any bearing on the addition of NFS?

Reply:
Reverting this before merging. I don't think these have any bearing on NFS addition, but I happened to notice slow performance with the default DB sizing (Basic 50), and somewhat inadvertently added this change in this PR, which I agree is not a good place for this change.